wire-ios/Wire-iOS Tests/SwiftSnapshotTesting/SnapshotTesting+UITableViewCell.swift

```diff
@@ -18,7 +18,7 @@
 
 import UIKit
 
-extension UITableViewCell: UITableViewDelegate, UITableViewDataSource {
+extension UITableViewCell: @retroactive UIScrollViewDelegate, @retroactive UITableViewDelegate, @retroactive UITableViewDataSource {
```

Comment:
what does @retroactive ?

Reply:
It silences warnings, however, I'm going to revert this.